lib/ast-processor.ts

```diff
@@ -10,22 +10,36 @@ import transformers, {
   mermaidTransformer,
   readmeComponentsTransformer,
   variablesTransformer,
+  handleMissingComponents,
 } from '../processor/transform';
 
 export interface MdastOpts {
   components?: Record<string, string>;
+  missingComponents?: 'ignore' | 'throw';
```

Comment:
Can you clarify the specific places in the app where we'd use one or the other? I imagine we'd `ignore` in view mode, but where would we `throw`?

Reply:
I was thinking it'd be useful in gitto, but realizing now, we'll probably do something else. I could revert all that.

Reply:
Well, actually, I'd like to keep it just in case.

Reply:
Mahesh, I can definitely imagine this coming in handy. But per our convo with @davinhazard (above), can we just make `ignore` the default?